src/canister/individual_user_template/src/data_model/mod.rs

```diff
@@ -82,34 +120,50 @@ pub struct CanisterData {
     pub watch_history: ic_stable_structures::btreemap::BTreeMap<WatchHistoryItem, (), Memory>,
     #[serde(skip, default = "_default_success_history_v1")]
     pub success_history: ic_stable_structures::btreemap::BTreeMap<SuccessHistoryItemV1, (), Memory>,
+    // u64 is post_id, SystemTime refers to time when first_bet is placed
+    #[serde(skip, default = "_default_bet_timer_posts_queue")]
+    pub bet_timer_posts:
+        ic_stable_structures::btreemap::BTreeMap<(SystemTimeInMs, PostId), (), Memory>,
+    // this keeps track of when the first bet was placed.
+    #[serde(skip, default = "_default_bet_timer_first_bet_placed_at_map")]
+    pub first_bet_placed_at_hashmap:
+        ic_stable_structures::btreemap::BTreeMap<PostId, (SystemTimeInMs, NewSlotType), Memory>,
```

Comment:
We can combine both of this into one. There is no need to have two maps if you think logically. Also we can store `PlaceBetArg, ()` directly.